web-client/src/presenter/state.ts

```diff
@@ -366,6 +366,7 @@ export const baseState = {
     qcSectionInProgressCount: number;
     qcIndividualInboxCount: number;
     qcIndividualInProgressCount: number;
+    unreadMessageCount?: number;
```

Comment:
Just curious about this

Reply:
`unreadMessageCount` is set sometimes on notifications state property. Its typed as an optional field here.